src/storage/redis_metadata.cc

```diff
@@ -329,7 +329,8 @@ bool Metadata::ExpireAt(uint64_t expired_ts) const {
 bool Metadata::IsSingleKVType() const { return Type() == kRedisString || Type() == kRedisJson; }
 
 bool Metadata::IsEmptyableType() const {
-  return IsSingleKVType() || Type() == kRedisStream || Type() == kRedisBloomFilter || Type() == kRedisHyperLogLog;
+  return IsSingleKVType() || Type() == kRedisStream || Type() == kRedisBloomFilter || Type() == kRedisHyperLogLog ||
```

Comment:
Wait, isn't a CMSketch a Single-KV Type?

Reply:
Oh, sorry about that, i was thinking due to the different variables, that it would be instead put under emptyable type. But after review it makes more sense as a singlekvtype